aea/cli/common.py

```diff
@@ -43,19 +43,24 @@
     ConfigurationType,
     ConnectionConfig,
     DEFAULT_AEA_CONFIG_FILE,
-    DEFAULT_PROTOCOL_CONFIG_FILE,
     Dependencies,
     ProtocolConfig,
     PublicId,
     SkillConfig,
     _get_default_configuration_file_name_from_type,
 )
 from aea.configurations.loader import ConfigLoader
+from aea.crypto.ethereum import ETHEREUM
 from aea.crypto.fetchai import FETCHAI
-from aea.helpers.base import (
-    add_agent_component_module_to_sys_modules,
-    load_agent_component_package,
+from aea.crypto.helpers import (
+    ETHEREUM_PRIVATE_KEY_FILE,
+    FETCHAI_PRIVATE_KEY_FILE,
+    _create_ethereum_private_key,
+    _create_fetchai_private_key,
```

Comment:
this could be simplified by using `FetchAICrypto.dump` and `EthereumCrypto.dump`, respectively

Reply:
https://github.com/fetchai/agents-aea/issues/881